patches/000-initial-setup.patch

```diff
@@ -674,14 +801,14 @@ index 51f9760187..93bbc26bd0 100644
  }
  
  func TestPSSSigning(t *testing.T) {
-+	if boring.Enabled {
++	if boring.Enabled && ! boringtest.Supports(t, "SHA1") {
```

Comment:
nit: Do you mind cleaning up the space in the negation operator? E.g. `!assertion()` as opposed to `! assertion()`.